api/net/inet.hpp

```diff
@@ -495,6 +514,7 @@ namespace net {
     std::string domain_name_;
 
     std::shared_ptr<net::DHClient> dhcp_{};
+    std::shared_ptr<net::Slaac>    slaac_{};
```

Comment:
Is this being shared between many Inet? If not it can just be a unique_ptr (if initialized on first use) or just a regular object.